packages/patternfly-4/react-core/src/experimental/components/OverflowMenu/OverflowMenuContentGroup.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/OverflowMenu/overflow-menu';
+import { OverflowMenuContentContext } from './OverflowMenuConstants';
+
+export interface OverflowMenuContentGroupProps extends React.HTMLProps<HTMLDivElement> {
+  /** Any elements that can be rendered in the menu */
+  children?: any;
+  /** Additional classes added to the OverflowMenuGroup */
+  className?: string;
+  /** Modifies the overflow menu group visibility */
+  persistent?: boolean;
+  /** Indicates a button or icon group */
+  groupType?: string;
```

Comment:
looks like it can be typed?
```suggestion
  groupType?:'button' | 'icon';
```